kratos/python_scripts/apply_periodic_boundaray_condition_process.py

```diff
@@ -0,0 +1,115 @@
+import KratosMultiphysics as KratosMultiphysics
+
+def Factory(settings, Model):
+    if(type(settings) != KratosMultiphysics.Parameters):
+        raise Exception("expected input shall be a Parameters object, encapsulating a json string")
+    return ApplyPeriodicBoundaryConditionProcess(Model, settings["Parameters"])
+
+## All the processes python should be derived from "Process"
+class ApplyPeriodicBoundaryConditionProcess(KratosMultiphysics.Process):
+    """This class is used in order to impose a rigid body movement in a certain region of the problem
```

Comment:
The comment seems wrong to me. It should not impose a rigid body movement, but rather a periodic boundary with a certain translation/rotation.

Reply:
done